index.html

```diff
@@ -173,6 +181,7 @@ <h2>Skills</h2>
     <img src="img/Icon1.png" alt="icon">
   </div>
   <div class="line"></div>
+  </section>
   </aside>
   <footer>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 1-Main (1)](https://user-images.githubusercontent.com/69638013/165756612-f77522a4-1f08-4c77-936f-c84bfe1e8ed2.png) | ![Screenshot_93](https://user-images.githubusercontent.com/69638013/165756657-ae58c722-7e0c-4c50-aacd-461b5f9ea0b2.png)

- [ ] From `768px` kindly increase width to look more like the figma design 👍🏿

Reply:
- [ ] [REQUESTED] Kindly make changes requested by the previous review